testworld/park_test.go

```diff
@@ -46,3 +46,70 @@ func TestHost_Happy(t *testing.T) {
 	getInvoiceAndCheck(eCharlie, docIdentifier, "GBP")
 	fmt.Println("Host test success")
 }
+
+func TestHost_AddExternalCollaborator(t *testing.T) {
```

Comment:
`park_test.go` is a sanity check and an example on the test framework it self, please move the test to a more relevant file eg: `document_update_test.go` (just a suggestion)

Reply:
good to know

Reply:
I should have added a comment, I know 😅